packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -27,20 +33,21 @@ const SEPOLIA_INFURA_RPC_URL = `https://sepolia.infura.io/v3/${INFURA_KEY}`
 export type ChainWithRpcUrl = Chain & {
   rpcUrl: string
   nativeTokenData?: Erc20Data
+  slug: string
```

Comment:
This should be optional

Reply:
done, now we also default query params for Orbit chains without slug to chain ID, so it works whether we set slug or not